packages/legacy/core/package.json

```diff
@@ -197,14 +199,14 @@
     "react-native-fs": "^2.16.6",
     "react-native-gesture-handler": "^1.10.3",
     "react-native-get-random-values": "^1.7.0",
-    "react-native-gifted-chat": "*",
```

Comment:
I can understand that `*` is super generic, but any particular reason for setting to a very specific peer dependencies? Usually keep compatibility at major level (`starting with `^`) is fine. Similar comment  for `react-native-qrcode-svg`, and `react-native-reanimated` changes below.

Reply:
The "*" caused installing latest stable version, and it caused a whole lot of conflicts with other packages that are mot updated, and to do that in this PR is a big mess. I had to fix the version to get things aligned together and get the app to build and tests to succeed.

Reply:
TBH, I believe leaving yarn to pickup latest updated versions for any packages is recipe for bugs. We have to dedicate PRs to solely update specific packages.

Reply:
we can be specific at `dependencies` but more liberal with `peer dependencies`. Also, dependencies should not be updated as a side-effect, it should be explicit and managed/lock by the lock file.

Reply:
keep in mind that `yarn install` MAY produce unexpected dependency change. `yarn install --immutable` is the safest command to use and void unexpected changes to the lock file

Reply:
I never meant to touch any of that :). It NEVER succeeded until I fixed versions for those packages. Let me know if you have another idea and we can hop on a call and discuss